styles/themes/charcoal.css

```diff
@@ -1174,6 +1174,10 @@ header #titles-preserved,
 header #logo,
 #header #logo {
   padding: 0.5em 0.5em 0.1em;
+  width: 360px;
+  height: 68px;
+  margin: 0.5em;
+  content: url('../../graphics/dp-logo-550x96px-light.png');
```

Comment:
Having the 500x96 for the branding page is great, but these files should remain 360x68 so we are not relying on the browser to resize them -- they'll look better across more browsers that way.

Reply:
For what it's worth, reverting to the 360x68 version of the logo looks *_much_* worse on Charcoal. Perhaps because it's "white" on "black", the fuzziness shows much more clearly than it does on the light themes.

Reply:
To clarify, is it that the larger image downsized in the browser was clearer, or was it that the larger image at actual size was clearer? In a prior rev the image was notably bigger than the current 360x68 and I don't think we want it to take up more room in the header. If it's truly the former then I'm amenable to having the larger images.

Reply:
Hm. I'm remembering that it was the larger image downsized in the browser was clearer. I had a browser window open with it using the 500px image inside the 360px container. Once this push came through, I refreshed the page. The image stayed the same size, but the quality degraded significantly. 

There won't be a difference on the light themes, but to me, the fuzziness is very noticeable on Charcoal.

Reply:
Erh ... to clarify, there won't be a difference on the light themes between this and the current live version. There was still a definite improvement with the 500px image in the 360px container on the light theme as well.

Reply:
Ohhhh. Bitten by the difference-in-browsers issues once again. I finally thought to look in Chrome (I had the page open from before), and indeed, when I refreshed the page, the image got smaller. I was just looking at it in Firefox. <sigh> And yes, it also looks clearer in the larger version, but is definitely too large.